src/app/components/team/SmoochBot/SmoochBotComponent.js

```diff
@@ -93,9 +93,9 @@ const SmoochBotComponent = ({
 
       Relay.Store.commitUpdate(
         new UpdateTeamBotInstallationMutation({
-          id: installation.id,
+          id: installation?.id,
```

Comment:
Does this mutation ever get called if there's no `installation`?

Reply:
It does not, so probably unnecessary. I'll remove the two checks here.